test/src/main/java/org/apache/accumulo/test/TableOperationsIT.java

```diff
@@ -59,20 +59,38 @@
 import org.apache.accumulo.core.security.TablePermission;
 import org.apache.accumulo.core.tabletserver.thrift.TabletClientService;
 import org.apache.accumulo.harness.AccumuloClusterHarness;
+import org.apache.accumulo.harness.MiniClusterConfigurationCallback;
+import org.apache.accumulo.harness.SharedMiniClusterBase;
+import org.apache.accumulo.minicluster.MemoryUnit;
+import org.apache.accumulo.minicluster.ServerType;
+import org.apache.accumulo.miniclusterImpl.MiniAccumuloConfigImpl;
 import org.apache.accumulo.test.functional.BadIterator;
 import org.apache.accumulo.test.functional.FunctionalTestUtils;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.RawLocalFileSystem;
 import org.apache.hadoop.io.Text;
 import org.junit.After;
+import org.junit.AfterClass;
 import org.junit.Before;
+import org.junit.BeforeClass;
 import org.junit.Test;
 
 import com.google.common.collect.Sets;
 
-public class TableOperationsIT extends AccumuloClusterHarness {
+public class TableOperationsIT extends SharedMiniClusterBase {
 
   static TabletClientService.Client client;
   private AccumuloClient accumuloClient;
 
+  @BeforeClass
+  public static void setupIT() throws Exception {
+    SharedMiniClusterBase.startMiniCluster();
```

Comment:
If we're not leveraging the callback, I'm not sure it makes sense to extend this.

Both SharedMiniClusterBase and ConfigurableMacBase are intended for configuring before running. ConfigurableMacBase is intended to start/stop on each test method, while SharedMiniClusterBase is intended to start/stop on the whole class... but otherwise, they serve the same role: a way to configure Mini in order to run an advanced test.

Almost all other tests are expected to run in the AccumuloClusterHarness, because they are tests which do not have this configurability requirement (beyond what the cluster harness implementations support, because they do support some configurability).